docs/book_files/csi_block_storage_kc_pdfs.md

```diff
@@ -0,0 +1,40 @@
+# Publications and related information
+
+The following Portable Document Format (PDF) publications and related online information sources are available for the IBM® block storage CSI driver.
+
+To view a PDF file, you need Adobe™ Reader. You can download it at no charge from the [Adobe website](http://get.adobe.com/reader/).
+
+## Publications
+
+|IBM block storage CSI driver publication|Description|PDF download link|
+|----------------------------------------|-----------|-----------------|
+|*Release notes*|This publication describes requirements, compatibility, change log, and known issues information for version 1.6.0.<br />First Edition (June 2021) for version 1.6.0.|![PDF icon](PDF_icon.jpg) [Download](../pdf/IBM_block_storage_CSI_driver_1.6.0_RN.pdf)|
+|*User guide*|This publication describes how to prepare for, install, configure, and use IBM block storage CSI driver.<br />Publication number: SC27-9590-10.|![PDF icon](PDF_icon.jpg)[Download](../pdf/IBM_block_storage_CSI_driver_1.6.0_UG.pdf)|
+
+## Related information and publications
+
+For more information about storage systems and orchestration platforms that are supported by IBM® block storage CSI driver, see the following resources.
+
+IBM resources
+
+-   [IBM SAN Volume Controller documentation](https://www.ibm.com/docs/en/sanvolumecontroller)
+-   [IBM Spectrum Scale documentation](https://www.ibm.com/docs/en/spectrum-scale)
+-   [IBM FlashSystem® 5200, 5000, 5100, Storwize® V5100 and V5000E documentation](http://www.ibm.com/docs/en/f555sv-and-v)
+-   [IBM FlashSystem™ 7200 and Storwize V7000 documentation](https://www.ibm.com/docs/en/flashsystem-7x00)
+-   [IBM Spectrum Virtualize as Software Only documentation](https://www.ibm.com/docs/en/spectrumvirtualsoftw)
+-   [IBM FlashSystem 9200 and 9100 documentation](https://www.ibm.com/docs/en/flashsystem-9x00)
+-   [IBM FlashSystem A9000 documentation](https://www.ibm.com/docs/en/flashsystem-a9000)
+-   [IBM FlashSystem A9000R documentation](https://www.ibm.com/docs/en/flashsystem-a9000r)
+-   [IBM DS8880 documentation](https://www.ibm.com/docs/en/ds8880)
+-   [IBM DS8900 documentation](https://www.ibm.com/docs/en/ds8900)
+-   [IBM Spectrum® Access for IBM Cloud® Private Blueprint](https://www-01.ibm.com/common/ssi/cgi-bin/ssialias?htmlfid=TSW03569USEN&)
+
+    Used as the FlexVolume driver based solution for OpenShift® 3.11, using [IBM Storage Enabler for Containers](https://www.ibm.com/docs/en/stgenablercontainers)
+
+-   [IBM Storage for Red Hat® OpenShift Blueprint](http://www.redbooks.ibm.com/abstracts/redp5565.html?Open)
+
+External resources
+-   [Persistent volumes on Kubernetes](https://kubernetes.io/docs/concepts/storage/volumes/)
+-   [Kubernetes Documentation](https://kubernetes.io/docs/home/)
+-   [Kubernetes Blog](https://kubernetes.io/blog/)
+
```

Comment:
This information is very similar to the [csi_relatedpubs.md](https://github.com/IBM/ibm-block-csi-driver/pull/320/files#diff-541cf3c47e7fd5270037dd506814bf87f9b42716f9aca6754e701fd5cf74466f) file.
Is it possible to remove the duplicated text and point to the other file instead?

Reply:
there's already [a conversation](https://github.com/IBM/ibm-block-csi-driver/pull/320#discussion_r628875238) about it